pyls/plugins/jedi_completion.py

```diff
@@ -45,8 +45,9 @@
 
 @hookimpl
 def pyls_completions(config, document, position):
+    settings = config.plugin_settings('jedi_completion', document_path=document.path)
     try:
-        definitions = document.jedi_script(position).completions()
+        definitions = document.jedi_script(position).completions(fuzzy=settings.get("fuzzy", False))
```

Comment:
you probably can check the version here with `jedi.__version__`

Although I am not sure what's the issue with bumping the jedi version for the release

Reply:
Maybe bumping Jedi's version is a bit.. rude, but I agree that bumping Jedi's should be done sooner or later.

Either way, i've updated the code to make sure we first check Jedi's version before enabling fuzzy completion and removed Jedi's version bump, see: 3ecff5f18854d6e6459184ef054d909029420c3b and 7e6d27a